packages/@webex/internal-plugin-metrics/src/call-diagnostic/config.ts

```diff
@@ -120,6 +125,8 @@ export const ERROR_DESCRIPTIONS = {
   DTLS_HANDSHAKE_FAILED: 'DTLSHandshakeFailed',
   ICE_FAILED_WITHOUT_TURN_TLS: 'ICEFailedWithoutTURN_TLS',
   ICE_FAILED_WITH_TURN_TLS: 'ICEFailedWithTURN_TLS',
+  SDP_OFFER_CREATION_ERROR: 'SdpOfferCreationError',
```

Comment:
might be worth adding a comment that this string has to match the name from SdpOfferCreationError class from internal-media-core